physionet-django/project/authorization/access.py

```diff
@@ -96,9 +96,6 @@ def can_access_project(project, user):
     if project.deprecated_files:
         return False
```

Comment:
Technically, the fact that files are deprecated shouldn't impact the fact that a user is authorized. It should impact the ability to download the files. I did not touch this to keep this as small as possible though.